src/tests/unit/tests/DetailsView/components/base-left-nav.test.tsx

```diff
@@ -7,11 +7,11 @@ import { Mock, MockBehavior } from 'typemoq';
 import { BaseLeftNav, BaseLeftNavLink, BaseLeftNavProps } from '../../../../../DetailsView/components/base-left-nav';
 
 class TestableBaseLeftNav extends BaseLeftNav {
-    public getOnNavLinkClick() {
+    public getOnNavLinkClick(): (event: React.MouseEvent<HTMLElement, MouseEvent>, item: BaseLeftNavLink) => void {
```

Comment:
`MouseEvent` again... do we need this here?

Reply:
I guess thats how a `React.MouseEvent` is defined and thats why vscode suggests me those. But fair point, we dont need them, so changed it.